<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WordleDialog</class>
 <widget class="QDialog" name="WordleDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>699</width>
    <height>453</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>370</y>
     <width>521</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTextEdit" name="opText">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>90</y>
     <width>331</width>
     <height>231</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="typedWords">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>90</y>
     <width>151</width>
     <height>101</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>50</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Words you typed:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>330</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Guess a word:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>60</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Output:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>671</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Rule of Wordle game</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>210</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Letter in th correct word:</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="pri">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>230</y>
     <width>171</width>
     <height>85</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
